package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "csurf",
   "description": "CSRF token middleware",
-  "version": "1.10.0",
+  "version": "1.11.0",
```

Comment:
Please remove this version bump from the PR, as we used npm to make the new releases and tags, which bumps and commits this for us.